gosling/display.py

```diff
@@ -13,68 +13,17 @@
 <html>
 <head>
   <style>.error { color: red; }</style>
-  <link rel="stylesheet" href="{{ higlass_css_url }}">
+  <link rel="stylesheet" href="{{ deps.css_url }}">
 </head>
 <body>
   <div id="{{ output_div }}"></div>
   <script type="module">
-
-    async function loadScript(src) {
-        return new Promise(resolve => {
-            const script = document.createElement('script');
-            script.onload = resolve;
-            script.src = src;
-            script.async = false;
-            document.head.appendChild(script);
-        });
-    }
-
-    async function loadGosling() {
-        // Manually load scripts from window namespace since requirejs might not be
-        // available in all browser environments.
```

Comment:
I assume we needed these part since Jupyter Lab/Notebook did not support requirejs and the use of esm.sh addresses this issue?